### ISSUE TITLE
Updated release for kodi 19.3

### ISSUE THREAD

**Issue by boxcar32** [oxagast/oxadio#4]
Hi,

I just found this after searching rtl-sdr for kodi. I had it working on Pi 400 Raspbian (buster) but sine installed libreElec and get "the python dependency xbmc.python 2.14.0 could not be satisfied". Just seeing if there will be an updated version.

Thanks,

**Comment by oxagast** [comment]
Yes, it will be when I find time to debug it, but I also don't have a Rasperry Pi 4 to test on that hardware specifically (may be able to find some type of emulator). I'm kinda tied up with other projects, and my day job currently.  Thanks for letting me know where it needs work!

**Comment by boxcar32** [comment]
Thanks for the reply.
